### PR TITLE
Explain details of "only" selector

### PR DESCRIPTION
**Pull request by jfehrle**
Add explanation related to https://github.com/coq/coq/issues/15441

FYI: @SkySkimmer

### CODE REVIEW

File: doc/sphinx/proof-engine/ltac.rst [hunk 1]
```diff
@@ -422,7 +422,11 @@ Selectors can also be used nested within a tactic expression with the
       range_selector ::= @natural - @natural
       | @natural
 
-   Applies :token:`ltac_expr3` to the selected goals.
+   Applies :token:`ltac_expr3` to the selected goals.  (At the beginning of a
+   sentence, use the form :n:`@selector: @tactic` rather than :n:`only @selector: @tactic`.
+   In the latter, the :opt:`Default Goal Selector` (commonly set to :n:`1:`)
```
Comment (SkySkimmer) [inline doc/sphinx/proof-engine/ltac.rst:427]:
```suggestion
   In the latter, the :opt:`Default Goal Selector` (by default set to :n:`1:`)
```

Comment (jfehrle) [reply doc/sphinx/proof-engine/ltac.rst:427]: That was my first thought, but I don't like repeating "Default ... default" in the sentence.

Comment (SkySkimmer) [reply doc/sphinx/proof-engine/ltac.rst:427]: "commonly" doesn't mean the same thing, that's more important than avoiding repetition

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
merging tomorrow

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**